taca/storage/storage.py

```diff
@@ -250,7 +255,8 @@ def _send_to_swestore(file_path, dest, remove=True):
             logger.info('Run {} successfully compressed! Removing from disk...'
                         .format(run))
             shutil.rmtree(run)
-            _send_to_swestore('{}.tar.bz2'.format(run), CONFIG.get('storage').get('irods').get('irodsHome'))
+            if not compress_only:
+                _send_to_swestore('{}.tar.bz2'.format(run), CONFIG.get('storage').get('irods').get('irodsHome'))
         else:
             logger.info("Run {} is not {} days old yet. Not archiving"
                         .format(run, str(days)))
```

Comment:
I mean `--force` should not check anything, right ? So line [244](https://github.com/guillermo-carrasco/TACA/blob/master/taca/storage/storage.py#L244) and [251](https://github.com/guillermo-carrasco/TACA/blob/master/taca/storage/storage.py#L251) would still perform the check even if `--force` is called, wouldn't it ??